Gemfile

```diff
@@ -117,4 +122,11 @@ group :test do
   # gem "capybara", "3.38.0"
   # gem "selenium-webdriver", "4.7.1"
   # gem "webdrivers", "5.2.0"
+  gem 'database_cleaner-active_record'
+  gem 'factory_bot_rails'
+  gem 'timecop'
 end
+
+gem 'bullet'
```

Comment:
very specific thing. I'm not going to install it to the common solution.
Great to have it as a PR, which could be merged at any time. Or to write an article about it.